site/book/07-effective-customizations/00.md

```diff
@@ -0,0 +1,17 @@
+Kubernetes configuration packages and customizations go hand in hand, all the 
+packaging tools enable package customization. In this chapter we cover effective 
+customizations techniques the kpt hydration and packaging enables.  We show how
+providing customization through parameters has some pitfalls and recommend
```

Comment:
I'd link to https://github.com/kubernetes/design-proposals-archive/blob/main/architecture/declarative-application-management.md#parameterization-pitfalls

Reply:
added